### DISCUSSION TITLE
Add "start mirroring" button to allow finding the page you want before displaying it

### DISCUSSION THREAD

**Discussion by e-Feitosa** [sircharlo/meeting-media-manager#3653] [Ideas] [answered]
### What do you want to discuss?

The feature "Add 'start mirroring' button to allow finding the page you want before displaying it" is already on the task list to be implemented in M3. **Is there an estimated time for this function to become available?** 
_I have been assisting some brothers, and I notice a great difficulty that some have in the week before the assembly/congress to show the assembly program. We know that we can download the program in PDF and add it to M3, which I personally find easy and quite interesting. However, if this feature to start mirroring the site on a specific screen becomes available, it can greatly facilitate this task for audio/video operators._

**A suggestion:** Upon finishing the site mirroring, it would be interesting for M3 to return to "Meeting Media".

![image](https://github.com/user-attachments/assets/ed7526df-1555-4d31-aa8e-524ae055c8a9)

**Comment by mtdvlpr** [comment]
I've starting working on this in #3758

**Comment by mtdvlpr** [comment, marked as answer]
This will be in the next release :)
